### PR TITLE
chore: move the required playwright to requirements.txt

### PR DESCRIPTION
**Pull request by voidking**
The main process depends on playwright.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -63,7 +63,7 @@ gitignore-parser==0.1.9
 websockets~=12.0
 networkx~=3.2.1
 google-generativeai==0.3.2
-# playwright==1.40.0  # playwright extras require
+playwright>=1.26 # used at metagpt/tools/libs/web_scraping.py
```
Comment (garylin2099) [inline requirements.txt:66]: 空两个#

File: setup.py [hunk 2]
```diff
@@ -24,7 +24,6 @@ def run(self):
 
 
 extras_require = {
-    "playwright": ["playwright>=1.26", "beautifulsoup4"],
```
Comment (garylin2099) [inline setup.py:27]: beautifulsoup不要了吗

Comment (voidking) [reply setup.py:27]: requirements.txt 中本身就有 beautifulsoup4，因此这里不需要了。

File: requirements.txt (hunk 1 above)
Comment (voidking) [reply requirements.txt]: 已增加空格

**Comment by garylin2099** [comment]
LGTM

**Pull request merged**